waves/src/App.tsx

```diff
@@ -297,6 +311,7 @@ function App() {
                                                         betaAmount,
                                                     },
                                                 })}
+                                            dataCy="exchange-asset-types"
```

Comment:
Let's have the naming convention 
`xx-button` if it is a button,
`xx-input` if it is an input field, 
`xx-textfield` if it is a textfield (read only). 

```suggestion
                                            dataCy="exchange-asset-types-button"
```